### PR TITLE
Fix golint issues in pkg kubelet container

### PR DESCRIPTION
**Pull request by obitech**
**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:
Fixes golinting issues in `pkg/kubelet/container` 
**Which issue(s) this PR fixes**:

Ref #68026 

**Special notes for your reviewer**:
Most of the changes are comments for exported functions. However, golint also complains about "stuttering" issues. For example types or functions of package `container` that contain `[cC]ontainer` in their name will result in a golint issue:

```
pkg/kubelet/container/runtime.go:179:6: type name will be used as container.ContainerID by other packages, and that stutters; consider calling this ID
```

I have tried to fix this in `pkg/kubelet/container/runtime.go` where it was not much effort, such as:

- `ContainerCommandRunner` -> `CommandRunner`
- `ContainerAttacher` -> `Attacher`

However this involved touching multiple other files.

Some of those issues I have not touched since I assume they will require a greater amount of files to be touched. In `pkg/kubelet/container/runtime.go`: 

- `ContainerID` -> `ID` ?
- `ContainerState` -> `State` ?
- `ContainerStatus` -> `Status` ? 

At the moment `hack/verify-golint.sh` is not passing because of these issues. What's the policy on these issues, should them be renamed?

**Does this PR introduce a user-facing change?**:

```release-note
Renaming of ContainerCommandRunner into CommandRunner and ContainerAttacher into Attacher
```

### CODE REVIEW

File: pkg/kubelet/container/testing/mockfileinfo.go [hunk 1]
```diff
@@ -27,83 +27,97 @@ import (
...
 type MockFileInfo struct {
 	ctrl     *gomock.Controller
-	recorder *_MockFileInfoRecorder
+	recorder *MockFileInfoRecorder
 }
 
-// Recorder for MockFileInfo (not exported)
-type _MockFileInfoRecorder struct {
+// MockFileInfoRecorder is a Recorder for MockFileInfo
+type MockFileInfoRecorder struct {
```
Comment (neolit123) [inline pkg/kubelet/container/testing/mockfileinfo.go]: should this be lowercase so that it's not exported?

**Review by neolit123 — COMMENTED** [review]
/priority backlog

**Comment by neolit123** [comment]
thanks for the cleanup PR @obitech

File: pkg/kubelet/container/testing/mockfileinfo.go (hunk 1 above)
Comment (obitech) [reply pkg/kubelet/container/testing/mockfileinfo.go]: Absolutely agree! Fixed it.

**Pull request closed without merging**